Applications/utils/npy_reader/npy_reader.cpp

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2024 Donghak Park <donghak.park@samsung.com>
+ *
+ * @file   npy_reader.cpp
+ * @date   28 Feb 2024
+ * @brief  reader for npy file
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author Donghak Park <donghak.park@samsung.com>
+ * @bug    No known bugs except for NYI items
+ */
+
+#include "npy_reader.h"
+
+#include <assert.h>
+#include <cstring>
+#include <iostream>
+#include <random>
+#include <string>
+#include <vector>
+
+namespace nntrainer::util {
+namespace {
```

Comment:
Do you really intend to have an unnamed namespace (not allowing other files to access) for a utility function?

Reply:
Fixed!